CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+## 0.61.1 / 2022-11-16
+
+* [CHANGE] Updated `RuleGroup` description and add validation for the CRD. #5072
+* [CHANGE] Removed validations in the operatior that are already covered at the CRD level. #5108
+* [CHANGE] jsonnet: Enforced existence of the TLS secret for the admission webhook deployment. #5112
+* [CHANGE] jsonnet: Changed default port of the admission webhook service from 8443 to 443. #5112
+* [CHANGE] Added a filter for non-running pods in the `ServiceMonitor` CRD. #5149
+* [FEATURE] Added `spec.attachMetadata.node` in the `ServiceMonitor` CRD. #5147
+* [ENHANCEMENT] Updated `ProbeTLSConfig` and `SafeTLSConfig` description. #5081
+* [ENHANCEMENT] Updated admission webhook deployment's jsonnet to avoid down-time on updates. #5099
+* [ENHANCEMENT] Added the `filterExternalLabels` field to the `Prometheus` CRD. #5142
+* [ENHANCEMENT] Added documentation for default monitor labels and how to configure them. #5136
+* [ENHANCEMENT] Added `enableHttp2` field to `AlertingEndpoints` #5152
+* [ENHANCEMENT] Updated `ThanosRuler` arguments (`QueryConfig`, `AlertManagerConfig`, `ObjectStorageConfig` and `TracingConfig`) to be directly read from secrets instead of using ENV vars. #5122
+* [BUGFIX] Updated incorrect configurations in `alerting.md`. #5102
+* [BUGFIX] Updated deprecated API usage in thanos readme. #5131
```

Comment:
I wonder about the need to add the previous two "bugfixes" which are docs/example updates?

Reply:
I had a similar thought, not sure if these are really bugs. In the case we want to have them, it could be useful to have a classification `[DOCS]`. Otherwise happy to remove them.

Reply:
Yeah, my thought process is that if we aren't shipping anything we should not include it but I am also ok with either model once we are consistent.

Any thoughts @simonpasquier

Reply:
Personally I don't include doc changes in the changelog. The categories (CHANGE, FEATURE, ...) were taken from the Prometheus project which doesn't list documentation updates.